tests/e2e/pact/specs/live/eps.convert.spec.ts

```diff
@@ -43,14 +46,16 @@ jestpact.pactWith(
 
         const requestStr = LosslessJson.stringify(request)
         const requestJson = JSON.parse(requestStr)
+        const requestId = uuid.v4().toString().toLowerCase()
 
         const apiPath = "/$convert"
         const interaction: InteractionObject = {
           state: "is authenticated",
           uponReceiving: `a request to convert ${desc} message`,
           withRequest: {
             headers: {
-              "Content-Type": "application/fhir+json; fhirVersion=4.0"
+              "Content-Type": "application/fhir+json; fhirVersion=4.0",
+              "X-Request-ID": requestId
```

Comment:
Maybe add `X-Request-ID` to the `willRespondWith` block as well to check it's present in the response. Could also test `X-Correlation-ID` in a similar way.

Reply:
Added